ios/AppLovinMAXAdView.m

```diff
@@ -0,0 +1,172 @@
+#import <AppLovinSDK/AppLovinSDK.h>
+#import "AppLovinMAX.h"
+#import "AppLovinMAXAdView.h"
+
+@interface AppLovinMAXAdView ()
```

Comment:
Not sure how you were developing these files, but I added them into the workspace.

![image](https://user-images.githubusercontent.com/5104410/188033751-edbbd2c5-d249-4b04-9c0e-88444b1fd39a.png)